menoh/tensorrt/model_core.cpp

```diff
@@ -41,10 +42,22 @@ namespace menoh_impl {
                     if(c.find("device_id") != c.end()) {
                         device_id = c["device_id"].get<int>();
                     }
+                    if(c.find("enable_profiler") != c.end()) {
+                        enable_profiler = c["enable_profiler"].get<int>();
+                    }
+                    if(c.find("allow_fp16_mode") != c.end()) {
+                        allow_fp16_mode = c["allow_fp16_mode"].get<int>();
+                    }
+                    if(c.find("force_fp16_mode") != c.end()) {
+                        force_fp16_mode = c["force_fp16_mode"].get<int>();
+                    }
```

Comment:
JSON has boolean values.
If `get<bool>` method exists, then I feel it is more appropriate than `get<int>`.

Reply:
Oh I didn't know it. I'll fix.